### ISSUE TITLE
Why Do Java Files Not Run If I don't have a folder open in VSCode

### ISSUE THREAD

**Issue by xFaust1** [redhat-developer/vscode-java#2242]
I am new to Java, and wanted to get started in VSCode. I ran my first Java file only to see that the terminal didn't open. Like it had the little run thingy on the top but the terminal didn't open and I couldn't see my file working.

I had happened to my .java file in some folder than uploaded that to VSCode, then ran that and that worked. Then I tried to run a .java file in VSCode even if it wasn't in the uploaded file, and that worked too. But when I close the uploaded folder and try to run the .java file and yeah it didn't work. Help please?

The code is something really simple. Like this for example.
```
public class stuff {
    public static void main(String[] args) {
        System.out.println("this is a test lol");
    }
}
```
##### Environment
- Operating System:  Windows
- JDK version: 17.0.1
- Visual Studio Code version: 1.62.3

**Comment by testforstephen** [comment]
Duplicates with https://github.com/redhat-developer/vscode-java/issues/2231, there is a regression about single file support (no folder opened in VS Code).

The CI build https://download.jboss.org/jbosstools/jdt.ls/staging/java-1.2.0-470.vsix has fixed it, you can [install from vsix](https://code.visualstudio.com/docs/editor/extension-marketplace#_install-from-a-vsix).

Another workaround is to open a folder containing your Java files on VS Code.

**Comment by rgrunber** [comment]
Closing as the duplicate issue was addressed. Should be fixed in 1.2.0.
